### DISCUSSION TITLE
v5.0 when?

### DISCUSSION THREAD

**Discussion by Melynt3** [capacitor-community/barcode-scanner#283] [General]
Hi, I just wanted to ask when we can expect the stable version 5.0, I admit that I checked the beta version, but I was a bit confused.
e.g. the whole layer of options like "hideBackground,showBackground" etc disappeared and dont really know how to handle this, im pretty new 😅 

Especially looking forward to zoom feature.

Thank you very much in advance for the information, you are doing a great job! o/

**Comment by Melynt3** [comment]
bump

**Comment by thegnuu** [comment]
Hi @Melynt3 

Sorry for not getting back to you sooner, unfortunately, I am currently too busy with other things to really maintain this repo.
At the current stage, I don't know if v5 will ever be released.

Our current plan is to join forces with https://github.com/capawesome-team/capacitor-mlkit/tree/main/packages/barcode-scanning and deprecate this repo in favor of the other implementation since it has overall more features and maintain the more or less same code base just does not make sense.

If you are looking for a working ml-kit version, please have a look at the project from capawesome!
I am not entirely sure if zooming is already supported, but it should not be that hard to add support if it is required.

I hope this helps!

**Comment by Melynt3** [comment]
Thanks a lot for reaching out! I completely understand your situation and appreciate your honesty about the current status of the repo. It's great to hear about your collaboration plans with the capawesome team's project. 

I'll definitely take a look at the project from capawesome for a working ml-kit version.

Thanks again for your response and the information, i also hopes it helps o/

Best regards
